pkg/cmd/server/start/start_master.go

```diff
@@ -242,10 +242,10 @@ func (o MasterOptions) RunMaster() error {
 			return err
 		}
 
-		if err := os.MkdirAll(path.Dir(o.MasterArgs.GetConfigFileToWrite()), os.FileMode(0755)); err != nil {
```

Comment:
@smarterclayton It looks like there are several examples of this being somewhat of a norm. /etc/grub.d and /etc/libvirt which are 0700 but files under them are more permissive. Apparently the reason that /etc/pki is open wider is that it's used by a wide variety of tools. Unless we're saying that something needs access to the ca.crt in /etc/origin/master I think we should tighten them. I don't think that we need access to that because it should be in the kubeconfig as well.